app/src/main/java/com/example/notes2text/usecases/FileEditingUseCases/EditTextInteractor.java

```diff
@@ -0,0 +1,36 @@
+package com.example.notes2text.usecases.FileEditingUseCases;
+
+
+import android.text.Spannable;
+
+
+public class EditTextInteractor implements EditTextBoundary{
+
+    private String content;
+
+    public EditTextInteractor(Spannable text) {
+        this.content = text.toString();
+    }
+
+    // Capitalize text selected by user
+    public String capitalize(int start, int end) {
```

Comment:
Make docstrings for all your public methods and public classes. Remember to also document your argument paramters.